mkdocs.yml

```diff
@@ -22,6 +22,8 @@ pages:
   - 'Using LibreTime':
     - 'On Air in 60 seconds!': 'manual/on-air-in-60-seconds/index.md'
     - 'Getting Started': manual/getting-started/index.md
+    - 'Tutorials' : manual/tutorials/index.md
```

Comment:
Just for consistency, can the `:` be directly after the key and the value be bounded by single quotation marks?